camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -517,13 +523,22 @@ def lrost(name, d, k, r, p, size):
         return calmessage
 
     @staticmethod
-    def lryaml(name, d, k, r, p, size):
+    def lryaml(name, d, k, r, p, size,dist_model):
         def format_mat(x, precision):
             return ("[%s]" % (
                 numpy.array2string(x, precision=precision, suppress_small=True, separator=", ")
                     .replace("[", "").replace("]", "").replace("\n", "\n        ")
             ))
 
+        # Select dist model
```

Comment:
You pass the distortion model in the function, you don't need these checks, just directly print.

Reply:
You need to find out the distortion model for formatting the message below

Reply:
Then you should create a function, right now you have the same block of code in 2 places